config/tailwind/v4.x.x/js-config/tailwind.config.js

```diff
@@ -0,0 +1,262 @@
+// Import GC Design System tokens
+import tokens from "./node_modules/@cdssnc/gcds-tokens/build/tailwind/tailwind.tokens.json";
```

Comment:
Same import clean up comment as above.

```suggestion
import tokens from "@cdssnc/gcds-tokens/build/tailwind/tailwind.tokens.json";
```